src/scripts/modules/form/addNewEmployee.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const { formatNumberToSalary } = require('../../utils');
+const { pushNotification, getErrors } = require('../notification');
+
+const createNewEmployee = (formData) => {
+  const tableRow = document.createElement('tr');
+
+  for (const [key, value] of Object.entries(formData)) {
+    const tableData = document.createElement('td');
+
+    let data = value;
+
+    if (key === 'salary') {
+      data = formatNumberToSalary(+data);
+    }
```

Comment:
```suggestion
    const salary = key === 'salary';
    const data = salary ? formatNumberToSalary(+data) : value;
```

Alternative variant.